packages/core/src/components/Form.js

```diff
@@ -332,6 +332,11 @@ export default class Form extends Component {
       const schemaValidationErrors = errors;
       const schemaValidationErrorSchema = errorSchema;
       if (Object.keys(errors).length > 0) {
+        const firstErrorElement = this.getFirstErrorField(
+          schemaValidation.errors[0]
+        );
+        firstErrorElement && firstErrorElement.focus(); // Focus on first error field
```

Comment:
firstErrorElement here could be either an Element _or_ a RadioNodeList

this would cover both use cases, focusing the element if it was one, OR the first item in the radio node list if it wasn't

```
        if (firstErrorElement instanceof Element) {
          firstErrorElement.focus();
        } else if (firstErrorElement instanceof RadioNodeList) {
          const firstItem = firstErrorElement.item(0);
          if (firstItem instanceof Element) {
            firstItem.focus();
          }
        }
```